### PR TITLE
#131356231 Game UI Redesign

### PR DESCRIPTION
**Pull request by andela-foladeji**
#### What does this PR do?

Improve the user interface of the game page

#### Description of Task to be completed?

When a user opts to start a new game, the user should see a redesigned interface to play the game

#### How should this be manually tested?

https://hyuga-cfh-staging-pr-41.herokuapp.com/

#### Any background context you want to provide?

N/A
#### What are the relevant pivotal tracker stories?

131356231 Re-Design UI for the actual gaming screen

#### Screenshots (if appropriate)

image

#### Questions:

N/A

### CODE REVIEW

File: public/views/answers.html [hunk 1]
```diff
@@ -1,89 +1,69 @@
...
+    <p>{{ makeAWishFact }}</p>
   </div>
-  <div id='charity-fact-container'>
-    <div id='charity-fact-tagline'>Did you know?</div>
+  
 
-    <div id='charity-fact'>
-       {{ makeAWishFact }}
-    </div>
+  <div ng-cloak ng-hide="game.state === 'game ended' || game.state ==='game dissolved' || game.state === 'awaiting players'">
```
Comment (neddinn) [inline public/views/answers.html]: You can wrap the condition in a function attached to a scope to make it more readable.

File: public/js/controllers/game.js [hunk 2]
```diff
@@ -158,27 +160,63 @@ angular.module('mean.system')
...
+                'color': 'black'
+              }).text(link);
             }, 200);
             $scope.modalShown = true;
           }
         }
       }
     });
 
+    $scope.makeDecision = function (state) {
```
Comment (neddinn) [inline public/js/controllers/game.js]: 👍

File: public/views/answers.html [hunk 3]
```diff
@@ -1,89 +1,65 @@
...
+        <li>Everyone else answers the black question card by clicking on the answer card they want to use.</li>
+        <li>The Card Czar then picks a favorite answer, and whoever played that answer wins the round.</li>
+      </ul>
     </div>
-    <div class="game-end-answer-text" ng-show="game.state === 'game ended' || game.gameWinner !== -1">
-      <h2>What would you like to do now?</h2>
+  </div>
+  <div ng-show="makeDecision('noGame')">
+    <h4 class="center-align">What would you like to do now?</h4>
+    <p class="center-align" ng-show="game.state ==='game dissolved' && game.gameWinner === -1">
```
Comment (neddinn) [inline public/views/answers.html]: Looks like you missed this. Might want to replace it with `makeDecision('gameEndPeopleLeft')`

File: public/views/answers.html [hunk 4]
```diff
@@ -1,89 +1,65 @@
...
+      You might consider donating to charity to atone for the horrifying answers you've submitted. Donating will also place a lovely
+      crown on your avatar's head.
+    </p>
+    <p class="center-align" ng-show="game.state === 'game ended' || game.gameWinner !== -1">
```
Comment (neddinn) [inline public/views/answers.html]: This too

File: public/views/answers.html (hunk 3 above)
Comment (andela-foladeji) [inline public/views/answers.html]: Ouch, I'll attend to them now

**Pull request merged**